### PR TITLE
Introduced yAxisWidth and related layout

### PR DESCRIPTION
**Pull request by djmiley**
To align with use and exposure of `xAxisHeight`

### CODE REVIEW

File: src/chart/linearTimeSeries.js [hunk 1]
```diff
@@ -4,6 +4,7 @@
     fc.chart.linearTimeSeries = function() {
 
         var xAxisHeight = 20;
+        var yAxisWidth = 40;
```
Comment (chrisprice) [inline src/chart/linearTimeSeries.js]: Can we leave the defaults as they were? i.e. 0

**Comment by chrisprice** [comment]
One small comment but otherwise :+1:

**Comment by ColinEberhardt** [comment]
Thanks for the contribution, I'm definitely :+1: to the idea of more configuration. Although I am wondering what this chart looks like if the y-axis width is != 0? Wouldn't you also have to change the y axis orientation (via `orient`) for this to make sense?

**Comment by chrisprice** [comment]
@ColinEberhardt I agree long-term it's probably not the right fix, but the same holds for the `xAxisHeight` so I think being consistent short-term is an ok fix.

**Comment by ColinEberhardt** [comment]
Far enough, if it fixes an immediate need, I'm happy :-)

**Comment by chrisprice** [comment]
Merged, thanks @djmiley

**Comment by djmiley** [comment]
:+1:

**Pull request merged**